ui/app/src/config.ts

```diff
@@ -0,0 +1,28 @@
+const config = {
```

Comment:
Add a comment to note that you should override this in your downstream implementations as needed